packages/components/package.json

```diff
@@ -78,6 +79,7 @@
     "@types/ember-qunit": "^6.1.1",
     "@types/ember-resolver": "^9.0.0",
     "@types/ember__destroyable": "^4.0.5",
+    "@types/luxon": "^3.2.0",
```

Comment:
I would suggest this get moved to `peerDependencies` so that applications can control what version they're on rather than bundling it with HDS. There's probably some opportunity to do this with other dependencies as well, including moving some to `optionalPeerDependencies`.

Reply:
Sorry I meant to suggest this about `luxon` not the types package 🤦.

Reply:
We use caret notations for all of our dependencies, allowing consumers to control what _minor_ or _patch_ version they're on without requiring two versions.

While using `peerDependencies` offers even more flexibility, allowing consumers to control which _major_ versions of certain libraries are used, it also introduces challenges around consumer experience, maintenance, and more importantly, potential compatibility issues.

Consumer experience:
 - The consumer will be responsible for installing a version that satisfies the peer dependency range. If they install (or already have installed) an incompatible version, it could lead to breakages, or the component may not work as expected
 - The consumer will have to manage the updates of peer dependencies themselves. This adds complexity, especially for less-experienced developers or those unfamiliar with your dependency requirements.
 - The out-of-the-box functionality currently offered by HDS components will be lost, offloading responsibility to the user.

Maintainer:
 - It becomes harder for the library to ensure consistent testing environments since the installed peer dependencies can vary between projects. We'll have to be meticulous in setting up tests for different versions of peer dependencies to cover potential version mismatches

For this specific component it means asking the majority of our consumers to install `luxon` if they want to use the Time component. HCP is using a compatible version, so no issues there. Atlas is using an incompatible version 2.x (with ReDoS vulnerabilities) so working with them to migrate to 3.x would be beneficial to everyone.